Intermittent integration test failure: test_promote_to_rollout

Seen here: https://app.circleci.com/pipelines/github/mozilla/experimenter/59142/workflows/0f7b31b9-4c45-4482-bdfe-f524a776ba17/jobs/347552

```py
_____________________ test_promote_to_rollout[FIREFOX_IOS] _____________________
[gw3] linux -- Python 3.12.3 /opt/venv/bin/python

selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="a32936f2-8281-4a52-a065-5cf84aaacfc8")>
create_experiment = <function create_experiment.<locals>._create_experiment at 0x7adad5520040>

    @pytest.mark.nimbus_ui
    @pytest.mark.xdist_group(name="group2")
    def test_promote_to_rollout(
        selenium,
        create_experiment,
    ):
        summary = create_experiment(selenium)
        summary.promote_first_branch_to_rollout()
>       summary.wait_for_clone_parent_link_visible()

experimenter/tests/integration/nimbus/test_experimenter_ui.py:61: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
experimenter/tests/integration/nimbus/pages/experimenter/summary.py:157: in wait_for_clone_parent_link_visible
    self.wait.until(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selenium.webdriver.support.wait.WebDriverWait (session="a32936f2-8281-4a52-a065-5cf84aaacfc8")>
method = <function presence_of_all_elements_located.<locals>._predicate at 0x7adad5522020>
message = 'Summary Page: could not find clone parent'

    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = "") -> T:
        """Wait until the method returns a value that is not False.
    
        Calls the method provided with the driver as an argument until the
        return value does not evaluate to ``False``.
    
        Parameters:
        -----------
        method: callable(WebDriver)
            - A callable object that takes a WebDriver instance as an argument.
    
        message: str
            - Optional message for :exc:`TimeoutException`
    
        Return:
        -------
        object: T
            - The result of the last call to `method`
    
        Raises:
        -------
        TimeoutException
            - If 'method' does not return a truthy value within the WebDriverWait
            object's timeout
    
        Example:
        --------
        >>> from selenium.webdriver.common.by import By
        >>> from selenium.webdriver.support.ui import WebDriverWait
        >>> from selenium.webdriver.support import expected_conditions as EC
    
        # Wait until an element is visible on the page
        >>> wait = WebDriverWait(driver, 10)
        >>> element = wait.until(EC.visibility_of_element_located((By.ID, "exampleId")))
        >>> print(element.text)
        """
        screen = None
        stacktrace = None
    
        end_time = time.monotonic() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, "screen", None)
                stacktrace = getattr(exc, "stacktrace", None)
            if time.monotonic() > end_time:
                break
            time.sleep(self._poll)
>       raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message: Summary Page: could not find clone parent

/opt/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:146: TimeoutException
- generated xml file: /code/experimenter/tests/integration/test-reports/experimenter_integration_tests.xml -
- Generated html report: file:///code/experimenter/tests/integration/test-reports/report.htm -
=========================== short test summary info ============================
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-5767)

--- a/experimenter/tests/integration/nimbus/pages/experimenter/summary.py
+++ b/experimenter/tests/integration/nimbus/pages/experimenter/summary.py
@@ -1,6 +1,3 @@
-import random
-import string
-
 from pypom import Region
 from selenium.webdriver.common.by import By
 from selenium.webdriver.common.keys import Keys
@@ -104,8 +101,6 @@ class SummaryPage(ExperimenterBase):
         By.CSS_SELECTOR,
         '[data-testid="label-release-date"]',
     )
-    _promote_to_rollout_form_locator = (By.CSS_SELECTOR, "#promoteToRolloutForm-control")
-    _promote_to_rollout_name_locator = (By.CSS_SELECTOR, "#promote-control-name")
     _promote_to_rollout_save_locator = (
         By.CSS_SELECTOR,
         "#promoteToRolloutForm-control .btn-primary",
@@ -294,15 +289,6 @@ class SummaryPage(ExperimenterBase):
         self.clone_name_field.send_keys(f"{text}")
 
     @property
-    def promote_to_rollout_name(self):
-        return self.wait_for_and_find_element(*self._promote_to_rollout_name_locator).text
-
-    @promote_to_rollout_name.setter
-    def promote_to_rollout_name(self, text=None):
-        el = self.wait_for_and_find_element(*self._promote_to_rollout_name_locator)
-        el.send_keys(f"{Keys.CONTROL}a", f"{text}")
-
-    @property
     def promote_to_rollout_save(self):
         return self.wait_for_and_find_element(*self._promote_to_rollout_save_locator)
 
@@ -320,10 +306,6 @@ class SummaryPage(ExperimenterBase):
 
     def promote_first_branch_to_rollout(self):
         self.js_click(self.promote_to_rollout_buttons[0])
-        random_chars = "".join(
-            random.choices(string.ascii_uppercase + string.digits, k=6)
-        )
-        self.promote_to_rollout_name = f"Rollout {random_chars}"
         self.js_click(self.promote_to_rollout_save)
 
     @property

